src/Dapr.Extensions.Configuration/DaprSecretStoreConfigurationSource.cs

```diff
@@ -24,6 +26,11 @@ public class DaprSecretStoreConfigurationSource : IConfigurationSource
         /// </summary>
         public IEnumerable<DaprSecretDescriptor> SecretDescriptors { get; set; }
 
+        /// <summary>
+        /// Gets or sets the secrets metadata.
```

Comment:
Can you copy a similar description for this from the updated version of DaprClient.cs?

Reply:
done